code/__HELPERS/grammar.dm

```diff
@@ -0,0 +1,37 @@
+/// Returns a list in plain english as a string
+/proc/english_list(list/input, nothing_text = "nothing", and_text = " and ", comma_text = ", ", final_comma_text = "" )
+	var/total = length(input)
+	switch(total)
+		if (0)
+			return "[nothing_text]"
+		if (1)
+			return "[input[1]]"
+		if (2)
+			return "[input[1]][and_text][input[2]]"
+		else
+			var/output = ""
+			var/index = 1
+			while (index < total)
+				if (index == total - 1)
+					comma_text = final_comma_text
+
+				output += "[input[index]][comma_text]"
+				index++
+
+			return "[output][and_text][input[index]]"
+
+/// Makes the first letter of a string to capital
+/proc/replace_first_letter_to_capital(string)
```

Comment:
do we not have some strings.dm helpers?

Reply:
we don't have such dm file. I think you're talking about `pronouns.dm` and that file only has verb transformation.

Reply:
found out it was in text.dm